### DISCUSSION TITLE
Internal Server Error 500 when checking out repo from classroom

### DISCUSSION THREAD

**Discussion by schrum2** [community/community#137643] [GitHub Education]
### Hi everyone,

I have several classes in GitHub classroom, and currently ALL of my links for accepting an assignment repo are resulting in the Internal Server Error 500 page. However, when I look at the repos in my organization, I am seeing each of the repos being created, even though none of them are visible within GitHub classroom under the particular assignment that has problems. I even have one example where a student checked out the assignment early, and her repo is visible in GitHub classroom, but other students cannot check it out. Something must be going wrong with GitHub today, since now everyone is getting the Internal Server Error 500, despite the fact that https://www.githubstatus.com/ claims everything is working.

A specific example repo:
https://classroom.github.com/a/h37lMsvj
Checking this repo out results in the Internal Server Error 500.

This is only one of several though, and as I said, checking out from these repos worked fine last week.

**Comment by okarpenko** [comment]
I am having the exact same problem.

**Comment by kanazuchijin** [comment]
Many of us are experiencing this same issue. See [this thread](https://github.com/orgs/community/discussions/137628) on the topic.

**Comment by schrum2** [comment]
Looks like GitHub fixed the issue
